src/main/scala/mesosphere/marathon/api/GroupApiService.scala

```diff
@@ -62,6 +62,10 @@ class GroupApiService(groupManager: GroupManager)(implicit authorizer: Authorize
       .getOrElse(createOrUpdateChange)
   }
 
+  def isAllowed[Resource](action: AuthorizedAction[Resource], mayResource: Option[Resource])(implicit identity: Identity, authorizer: Authorizer): Boolean = {
+    checkAuthorizationOrThrow(action, mayResource.get)(identity, authorizer)
+  }
```

Comment:
this may seem unusual.. however the original code below `checkAuthorizationOrThrow` was called by 2 functions... 1 was line 55 which is for a Group.   The other was line 36 for an `Option[Group]' which didn't work

Reply:
there is probably a more "clever" way to solve this... whatever it is... lets do it

Reply:
Thanks for fixing this!! Not sure my way is more "clever" but check it out and tell me what you think... :)